deploy/helm/templates/deployment.yaml

```diff
@@ -52,6 +53,10 @@ spec:
             {{- with .Values.loggerSettings.encoder }}
             - "--zap-encoder={{ . }}"
             {{- end }}
+            - "--extensions={{- default "true" ( include "kubeblocks.addonControllerEnabled" . ) }}"
+            - "--apps=true"
+            - "--workloads=true"
+            - "--dataprotection=false"
```

Comment:
should isolation deployment be a switch, i.e., enabled  + separatedDeployment?

Reply:
Done. added `separatedDeployment` helm value.